specifyweb/frontend/js_src/lib/components/FormCells/FormTable.tsx

```diff
@@ -189,6 +196,20 @@ export function FormTable<SCHEMA extends AnySchema>({
     scrollerRef
   );
 
+  const resourceHasBlockersContext = React.useContext(HasBlockersContext);
```

Comment:
Why not simplify it?

```tsx
const hasBlockers = React.useContext(HasBlockersContext);
React.useEffect(()=>{
  if(hasBlockers) handleExpand();
},[hasBlockers, handleExpand]);
```

Reply:
Even more, you don't need context in this case (Though I do like that you know how to use it).

Simply add a `parentResource` prop to this component and then you can get hasBlockers like this:

```tsx
const blockers = useAllSaveBlockers(parentResource, relationship);
const hasBlockers = blockers.length > 0;
```